src/Features/CSharp/Portable/BraceCompletion/StringLiteralBraceCompletionService.cs

```diff
@@ -88,7 +88,7 @@ protected override Task<bool> IsValidOpenBraceTokenAtPositionAsync(SyntaxToken t
 
         private static bool RestOfLineContainsDiagnostics(SyntaxToken token)
         {
-            while (!token.TrailingTrivia.Contains(t => t.IsEndOfLine()))
+            while (token != default && !token.TrailingTrivia.Contains(t => t.IsEndOfLine()))
```

Comment:
Comparison to `default` requires checking a bunch of unnecessary fields and is noticeably slow on some hot paths (e.g. in formatter). The fix is to use this equivalent:

```suggestion
            while (!token.IsKind(SyntaxKind.None) && !token.TrailingTrivia.Contains(t => t.IsEndOfLine()))
```